### ISSUE TITLE
sounds

### ISSUE THREAD

**Issue by fenwickjb** [CompassSoftware/AppTAG#169]
We need to make sure the "correct" and "wrong" sounds are playing for the activities.

Firefox was complaining about the wrong.wav file not being supported. Can we find a different one? The correct sound is being heard in some activities, but not all.

Also, the background sound playing? Sometimes I hear and sometimes I don't. Not sure if it's a browser thing or not. We need to at least look into this...

**Comment by JerichoSM** [comment]
Notes:
-FireFox Web Dev only complains about the "wrongAnswer.wav" out of the three sounds.
-FireFox (and possibly other browsers) sometimes autoblocks any sound, have to give the web page permissions to play the music. 
 -- "correctAnswer" plays even with the blocked audio sometimes?
-Changing the .wav into a .mp3 did not work to fix the issue, but the Web Dev message changed.
 -- FireFox Web Dev states the "wrongAnswer.mp3" cannot be decoded.

Potential Solution, find a new sound byte and download it in MP3 and test.
OR
Find an actual .wav to .mp3 converter, not just renaming the file and test.

**Comment by fenwickjb** [comment]
I think find a new sound file.
In Safari and Chrome, I do hear the wrong.wav sound play. 
According to this W3Schools page on <audio> tag, Firefox should be able to play a wav but IE can't. So, I think let's just find another sound file for that one.
https://www.w3schools.com/tags/tag_audio.asp

And more data:
From the Chrome developer debug panel, I got a message about the background music (I heard it the first time) when I refreshed. The error was: 
phaser.js:78728 The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. 
https://goo.gl/7K7WLu
And, it had a link to explain the problem, which is below.
https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#webaudio
